.env

```diff
@@ -1 +1 @@
-NEXT_PUBLIC_SIMPLEJSONCMS_ID=mdb18slfe7
+SIMPLEJSONCMS_ID=mdb18slfe7
```

Comment:
I think we should call this `CHARACTERS_SIMPLEJSONCMS_ID` because we'll probably have more than one.